core/src/main/scala/com/softwaremill/react/kafka/ReactiveKafka.scala

```diff
@@ -95,15 +98,47 @@ class ReactiveKafka {
     Sink.fromGraph(new KafkaGraphStageSink(producer))
   }
 
+  // Scala DSL
+  def graphStageSource[K, V](
+    props: ConsumerProperties[K, V],
+    topicsAndPartitions: List[TopicPartition] = List(),
```

Comment:
I was just making use of the `TopicPartitionOffset` case class suggested by @kciesielski earlier.